ika/client_mock.go

```diff
@@ -0,0 +1,26 @@
+package ika
+
+import (
+	"context"
+
+	"github.com/block-vision/sui-go-sdk/models"
+	tmtypes "github.com/cometbft/cometbft/types"
+)
+
+// MockClient is a mock implementation of the Client interface for testing.
+type MockClient struct{}
```

Comment:
let's move this file to `client_mock_test.go` or to another package

Reply:
This is used in different packages for testing. We cannot have it in `_test.go` because it will no be visible for other packages